scanner/src/main/kotlin/Scanner.kt

```diff
@@ -192,7 +201,7 @@ class Scanner(
                 }.onFailure {
                     controller.addProvenanceResolutionIssue(
                         pkg.id,
-                        Issue(source = TOOL_NAME, severity = Severity.ERROR, message = it.collectMessages())
+                        Issue(source = TOOL_NAME, message = it.collectMessages())
```

Comment:
this is done in wrong commit.

Reply:
What is the rationale for this? The default might change, but this would still be an error.

Reply:
@sschuberth was asking in this review to not pass redundant default values. IIRC rationale was consistency.